### PR TITLE
Changes to donation form, plus a simple donation review page

### PR DESCRIPTION
**Pull request by khandelwal**
This is a larger pull request than I would normally submit. Mostly, I forgot there were all these changes. 

This adds some missing fields to the donation form, and makes an attempt to break apart individuals and organizations. I'm sure in the future those two entities will see different fields too. 

Happy to answer questions, and look forward to the code review.

### CODE REVIEW

File: peacecorps/peacecorps/forms.py [hunk 1]
```diff
@@ -1,19 +1,15 @@
 from django import forms
+
 from localflavor.us.forms import USStateField
 from localflavor.us.forms import USStateSelect
+from django_countries.widgets import CountrySelectWidget
```
Comment (cmc333333) [inline peacecorps/peacecorps/forms.py]: Doesn't look like this gets used, and I don't think it's part of requirements.txt?

File: peacecorps/peacecorps/tests/test_forms.py [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
+            'street_address': '1 Main Street',
+            'city': 'Anytown',
+            'country': 'USA',
+            'state': 'MD',
+            'zip_code': '20852',
+            'payment_type': 'credit-card',
+            'information_consent': 'vol-consent-yes'
+        }
+        form = OrganizationDonationForm(data=form_data)
+        print(form.errors)
```
Comment (cmc333333) [inline peacecorps/peacecorps/tests/test_forms.py]: Stay debug statement

File: peacecorps/peacecorps/views.py [hunk 3]
```diff
@@ -1,15 +1,90 @@
...
+    memo += phone
+    memo += '(yes)(no)(yes)'
+    return memo
+
+
+def donation_payment_review(request):
+    """ This view is for a simple donation payment review page. """
+    data = {}
+    for k, v in request.session.items():
+        data[k] = v
```
Comment (cmc333333) [inline peacecorps/peacecorps/views.py:78]: Ultimately, this needs to be saved into a model if it needs to be passed to pay.gov

**Comment by cmc333333** [comment]
Any chance you can setup Travis? I think it would have caught the missing requirement.

File: peacecorps/peacecorps/views.py (hunk 3 above)
Comment (khandelwal) [inline peacecorps/peacecorps/views.py:78]: True. I avoided that for now in the hopes that we can store less and less on our end.

**Comment by khandelwal** [comment]
Ok, I think I've setup Travis. We'll see once this is merged right? 
Also, all other comments resolved.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+    - "3.4"
+    install:
+        - pip install -r requirements.txt --use-mirrors
+    script:
+        - python manage.py test --settings=peacecorps.settings.test
```
Comment (cmc333333) [inline .travis.yml:7]: Whoops. Missed this before merging: lines 4 through 7 should all be outdented once, right?

Comment (khandelwal) [inline .travis.yml:7]: Try: https://github.com/18F/peacecorps-site/pull/5

**Pull request merged**